lib/node_runner_checker.go

```diff
@@ -420,7 +420,7 @@ func SIGNBallotBroadcast(c sebakcommon.Checker, args ...interface{}) (err error)
 	var runningRound *RunningRound
 	var found bool
 	if runningRound, found = rr[checker.Ballot.Round().Hash()]; !found {
-		err = errors.New("RunningRound not found")
+		err = stderrors.New("RunningRound not found")
```

Comment:
If we provide `New` in `errors`, as @anarcher suggested, this becomes unnecessary.

Reply:
@Geod24 @anarcher Now I got it. Thanks :)